compiler/hash-pipeline/src/fs.rs

```diff
@@ -50,7 +49,7 @@ impl Display for ImportErrorKind {
 #[derive(Debug, Clone)]
 pub struct ImportError {
     /// The path that was attempted to be imported.
-    pub path: InternedStr,
+    pub path: String,
```

Comment:
use `AllocId`?

Reply:
Would be nice to have a newtype wrapper around AllocId for strings

Reply:
like `struct StringId(AllocId)`

Reply:
Now using `StringId`